03_launch_mgmt_cluster.sh

```diff
@@ -225,11 +191,29 @@ EOF
   else
     # Deploy Ironic using deploy.sh script
 
-    # Update container images to use local ones
-    cp "${BMOPATH}/ironic-deployment/ironic/ironic.yaml" "${BMOPATH}/ironic-deployment/ironic/ironic.yaml.orig"
-    cp "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml" "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml.orig"
-    update_kustomization_images "${BMOPATH}/ironic-deployment/ironic/ironic.yaml"
-    update_kustomization_images "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml"
+    cp "${BMOPATH}/ironic-deployment/default/kustomization.yaml" "${BMOPATH}/ironic-deployment/default/kustomization.yaml.orig"
+    cp "${BMOPATH}/ironic-deployment/keepalived/kustomization.yaml" "${BMOPATH}/ironic-deployment/keepalived/kustomization.yaml.orig"
+
+    if [ -n "${IRONIC_LOCAL_IMAGE}" ]; then
+      update_component_image Ironic "${IRONIC_LOCAL_IMAGE}"
+    else
+      update_component_image Ironic "${IRONIC_IMAGE}"
```

Comment:
do we really needs else part? I assume if we drop it, we will anyways use the latest images or the image as defaulted in `IRONIC_IMAGE`?

Reply:
We need both. The `*_LOCAL_IMAGE` variables are for images that are built locally while the other are for "the rest". In CI we use them to set the registry for example so we can use nordix harbor instead of docker hub or quay. For some tests we also want specific tags, like when upgrading from an older version.